### ISSUE TITLE
Leaderboard

### ISSUE THREAD

**Issue by LMPerson** [brokentowel/CSCE121-Group43#1]
The leaderboard file seems to have a lot of errors, so I'm testing out this issue thing guise.

**Comment by brokentowel** [comment]
yeah, it's mostly pseudo/placeholder code until i have time to debug
